packages/core/src/extensions/Blocks/nodes/Block.ts

```diff
@@ -46,38 +43,34 @@ declare module "@tiptap/core" {
 export const Block = Node.create<IBlock>({
   name: "block",
   group: "block",
+
+  // Ensures content-specific keyboard handlers trigger first.
+  priority: 50,
+
   addOptions() {
     return {
       HTMLAttributes: {},
     };
   },
 
   // A block always contains content, and optionally a blockGroup which contains nested blocks
-  content: "content blockgroup?",
+  content: "(textBlock | headingBlock | listItemBlock) blockgroup?",
```

Comment:
I think it's good to keep referring to blocks the same way we do now (i.e. the nodes which have drag handles, ids, etc) and change the naming of the inner blocks (previously block content) to something else since now they're also called xyzBlock which is confusing.